bin/add_transform_test.py

```diff
@@ -45,6 +47,20 @@ def add_regional_endpoint_configuration_if_needed(template: Dict[str, Any]) -> D
 
     return template
 
+def update_partition(region: str, template: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Since we actually doing smth here and in `generate_transform_test_output_files` it'd be great to have some tests

Reply:
hmm it's a script to run the translator given a file and move those files to related folder. Unit tests seems useless and integration test is a overkill because to really see the desired file, we have to set region to cn or us-gov. So I don't find a meaningful testing solution here.

Reply:
Right, that's the script!